client/src/document/ingredients/prose.tsx

```diff
@@ -1,4 +1,4 @@
-import { DisplayH2, DisplayH3 } from "./utils";
+import { DisplayHeading } from "./utils";
```

Comment:
This file imports `DisplayHeading`, but doesn't use it yet. Essentially this is causing the build failure.